tests/Rules/ProtectedClassMethodInFinalClassRule/Fixture/ProtectedMethodInFinalClass.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TomasVotruba\UnusedPublic\Tests\Rules\ProtectedClassMethodInFinalClassRule\Fixture;
+
+final class ProtectedMethodInFinalClass
+{
+    public function publicMethod()
+    {
+    }
+
+    public static function staticPublicMethod()
+    {
+    }
+
+    protected function protectedMethod()
```

Comment:
seems like there is a ECS style rule which reduces the visibility of the method and therefore destroys the test-case :)

Reply:
skipped with [4814f0e](https://github.com/TomasVotruba/unused-public/pull/68/commits/4814f0e3be07ab5e8d2c8854d279b4030a6294f9)